x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -11,14 +11,14 @@ import type { CommonProps } from '@elastic/eui';
 import { EuiFlexGroup, EuiFlexItem, useResizeObserver, EuiButtonIcon } from '@elastic/eui';
 import styled from 'styled-components';
 import classNames from 'classnames';
+import { EnteredInput } from './lib/entered_input';
+import type { InputCaptureProps } from './components/input_capture';
+import { InputCapture } from './components/input_capture';
```

Comment:
Since the `InputCaptureProps` and `InputCapture` are imported from the same path you can also do

`import { type InputCaptureProps, InputCapture } from ...`

Reply:
Just noticed that pattern recently. I normally just let the ESLint rule handle it when I save the file, but if I remember in the future, I'll use it.